creating_images/guidelines.adoc

```diff
@@ -273,9 +273,23 @@ that a process which is able to escape the container due to a vulnerability in
 the container framework will not have specific user permissions on the host
 system.
 
-To accomplish this, directories that must be written to by processes in the
-image should be world-writable. In addition, the processes running in the
-container must not listen on privileged ports (ports below 1024).
+To support running as an arbitray user, directories and files that must be written to by 
```

Comment:
I don't think this is the only option to do this.  So maybe change the wording to list it as an option.

https://docs.openshift.com/enterprise/3.2/install_config/persistent_storage/pod_security_context.html#supplemental-groups
https://docs.openshift.com/enterprise/3.2/install_config/persistent_storage/pod_security_context.html#fsgroup